atoms/Cargo.toml

```diff
@@ -1,10 +1,12 @@
 [package]
 name = "servo_atoms"
-version = "0.0.1"
+version = "0.1.0"
```

Comment:
What's the reasoning for the version bump here?

Reply:
No particular reason other than it's usual for the first published version of a crates to be `0.1.0`. That allows for the possibility of non-breaking releases (I believe otherwise Cargo will treat the patch version as breaking).